wolfcrypt/src/asn.c

```diff
@@ -1116,6 +1116,98 @@ static int GetASN_BitString(const byte* input, word32 idx, int length)
     return 0;
 }
 
+/* Check a UTF8STRING's data is valid.
```

Comment:
Should we have a size reduction option for this new check? Perhaps wrap in `#ifndef WOLFSSL_NO_ASN_STRICT`?

Reply:
Agreed.
Changed as requested.